archey/entries/uptime.py

```diff
@@ -8,41 +17,147 @@ class Uptime(Entry):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        with open('/proc/uptime') as file:
-            fuptime = int(file.read().split('.')[0])
+        uptime_seconds = int(self._get_uptime_delta().total_seconds())
```

Comment:
I took the liberty to transform your methods as static, hope you don't mind.

Reply:
Yeah, that looks more sensible - the flow is a lot easier to follow like this. Nice!